utils/tests/utils.go

```diff
@@ -82,6 +82,17 @@ func removeDirs(dirs ...string) {
 	}
 }
 
+// After syncDeletes we must make sure that there are no actual files we did not expect.
+func CheckSyncedDirContent(expected, actual []string, t *testing.T) {
```

Comment:
Looks like this function is used only from *artifactory_test.go*. Any reason why not move it there?
It is very specific for the Artifactory upload and download commands. It is currently inside a generic test util file.